### PR TITLE
Add transition and max-width helpers

### PR DESCRIPTION
**Pull request by ltclmbdev**

### CODE REVIEW

**Comment by ltclmbdev** [comment]
Add transition and max-width helpers

**Comment by ltclmbdev** [comment]
Joel, Trevor - are you ok with `eh-` prefix for this sort of stuff?

File: src/tachyons-base.css [hunk 1]
```diff
@@ -31,3 +31,14 @@ a {
 code {
   color: #59cd90; /* green */
 }
+
+
+/* Max-width */
+.eh-mw9 {
+  max-width: 76rem;
+}
```
Comment (trevordmiller) [inline src/tachyons-base.css:39]: Seems that this should go with the other max width classes:
image

File: src/tachyons-base.css [hunk 2]
```diff
@@ -31,3 +31,14 @@ a {
...
+
+/* Transitions */
+.eh-transition {
+  transition: all 150ms ease-in-out;
+}
```
Comment (trevordmiller) [inline src/tachyons-base.css:44]: Seems that this should go with the other animation related classes:
image

**Comment by trevordmiller** [comment]
I'd prefer just keeping everything together as Tachyons (so for example, if you are going to overwrite the default `mw9` just overwrite it in place rather than adding a prefix); but why is there a need to overwrite the dfeault `mw9`? That will break Tachyons relative number system.

**Comment by ltclmbdev** [comment]
About `.mw9`: our design has 1180px width for content. I have rounded it to 1184 (74rem). There is no tachyons native class to fit our needs, so we definitely need the custom max-width helper (as I understand the things). Override tachyons `.mw9` is an option, but I decided to create the custom one (it is discussible for sure!). I choosed the `.eh-mw9` to make it clear to understand what this class relates to.
Actually, the same reasons for `.eh-transition`. 
Lets define the tachyons customization strategy, so we can apply changes in same, defined by all of us, way. Make sense?

**Comment by ltclmbdev** [comment]
I said content is 1184px (74rem), but I applied 76rem for this class to have ability to add left&right paddings. Something like Twitter Bootstrap's `.container` class

**Comment by trevordmiller** [comment]
Ok, whatever works. I would recommend keeping all the related overrides in the related tachyons repo though as mentioned above rather than in tachyons-base. But its up to you and @tayiorbeii

**Comment by joelhooks** [comment]
went another direction with this.

**Pull request closed without merging**